uwblueprint/childrens-aid-society
# Fail linting GH action when frontend throws warnings

## Notion ticket link

[Investigate frontend linting GH action](https://www.notion.so/uwblueprintexecs/Investigate-frontend-linting-GH-action-2b04a65df66e4430800405aba87c1c5f)



## Implementation description
* Change linting action so warnings cause GH action to fail (currently they just show up as warnings but aren't blockers to merging)



## Steps to test
1. Verify that [this build](https://github.com/uwblueprint/childrens-aid-society/runs/7551786475?check_suite_focus=true) failed due to linting issue



## What should reviewers focus on?
* Does this work?


## Checklist
